cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc

```diff
@@ -1,9 +1,9 @@
 .User management command syntax
-[options="header", width="100%", cols="5a,3,2,^,^."]
+[options="header", width="100%", cols="7a,3,2"]
 |===
-| Command | Description | Required privilege | Community Edition | Enterprise Edition
```

Comment:
here were 2 columns? I know we somewhere have one but yes, I agree that we remove them here